.npmrc

```diff
@@ -0,0 +1,4 @@
+# Delete this .npmrc file to use the npmjs registry instead of the Microsoft Azure Artifacts feed for local development
```

Comment:
Do we think we should put something referring only to OSS at the beginning of this comment? 
```suggestion
# [OSS Contributors] Delete this .npmrc file to use the npmjs registry instead of the Microsoft Azure Artifacts feed for local development
```

Reply:
yeah this makes sense